home/templates/home/about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
Follow the format from other html file. You dont have to define header and footer. Just look at other files as examples.